src/allmydata/test/common_util.py

```diff
@@ -87,13 +87,18 @@ def run_cli_native(verb, *args, **kwargs):
     argv = nodeargs + [verb] + list(args)
     stdin = kwargs.get("stdin", "")
     if encoding is None:
-        # The original behavior, the Python 2 behavior, is to accept either
-        # bytes or unicode and try to automatically encode or decode as
-        # necessary.  This works okay for ASCII and if LANG is set
-        # appropriately.  These aren't great constraints so we should move
-        # away from this behavior.
-        stdout = StringIO()
-        stderr = StringIO()
+        if PY2:
```

Comment:
I believe this makes Python 3 CLI tests match the behavior of Python 3 in the real world, with sys.stdout and stderr always being unicode.